human-biomolecular-data/data-analysis.md

```diff
@@ -19,11 +25,111 @@ training:
+    - *Phylogenetic analysis*: [ClustalW](https://github.com/coldfunction/CUDA-clustalW), [MUSCLE](https://github.com/rcedgar/muscle), [MAFFT](https://github.com/GSLBiotech/mafft) and [PhyML](https://github.com/stephaneguindon/phyml)
+    - *Variant calling*: [Dragen-GATK](https://gatk.broadinstitute.org/hc/en-us/articles/360045944831), [DeepVariant](https://github.com/google/deepvariant), [FreeBayes](https://github.com/freebayes/freebayes) and [VarScan](https://github.com/dkoboldt/varscan)
+    - *Annotation*: [ANNOVAR](https://github.com/WGLab/doc-ANNOVAR), [SnpEff](https://github.com/pcingola/SnpEff), [VEP](https://github.com/Ensembl/ensembl-vep) and [dbNSFP](https://github.com/shiquan/bcfanno/blob/master/Documentation/database/dbNSFP.md)
+- **Epigenetic analysis**: This involves measuring changes in DNA methylation, histone modifications, or other epigenetic marks in different samples or conditions. This can help to understand how gene expression is regulated and identify potential biomarkers or therapeutic targets.
+    - *DNA methylation analysis*: [Bismark](https://github.com/FelixKrueger/Bismark), [MethylKit](https://github.com/al2na/methylKit) and [methylPipe](https://bioconductor.riken.jp/packages/3.1/bioc/html/methylPipe.html)
+    - *Histone modification analysis*: [MACS](https://github.com/macs3-project/MACS) and [SICER2](https://github.com/zanglab/SICER2)
+- **Protein-protein interaction analysis**: This involves identifying proteins that interact with each other and exploring the functional consequences of these interactions. This can help to identify new targets for drug development and understand disease mechanisms.
+    - *Interaction databases*: [BioGRID](https://github.com/BioGRID) and [IntAct](https://github.com/intact-portal)
+    - *Network analysis*: [Cytoscape](https://apps.cytoscape.org/) and [GeneMANIA](https://github.com/GeneMANIA/genemania)
+- **Metabolomics analysis**: This involves measuring the levels of small molecules (metabolites) in biological samples and comparing them across different conditions or groups of samples. This can help to identify biomarkers of disease or drug response.
+    - *Data processing*: [XCMS](https://xcmsonline.scripps.edu/landing_page.php?pgcontent=mainPage), [MZmine](http://mzmine.github.io/) and [OpenMS](https://github.com/OpenMS/OpenMS)
+    - *Statistical analysis*: [MetaboAnalyst](https://www.metaboanalyst.ca/) and [MetSign](https://pubmed.ncbi.nlm.nih.gov/21932828/)
+
+## Postprocessing
+
+The postprocessing part refers to the steps taken after the initial analysis to refine and interpret the results. Post-processing steps are important because they can help to identify biological patterns and relationships that were not apparent in the initial analysis, and to ensure that the results are biologically meaningful and reproducible.
```

Comment:
Tiny detail, in text here says 'Post-processing' while title says 'Postprocessing' :)

Reply:
This change has been addressed